boards/arm/arduino_zero/doc/arduino_zero.rst

```diff
@@ -120,8 +120,6 @@ Flashing
 
 #. To flash an image:
 
-   .. code-block:: console
-
 	.. zephyr-app-commands::
```

Comment:
You should also outdent this ``.. zephyr-app-commands::`` directive too (align with the T in "To flash")
(the same for the directive on line 97 above)